packages/design-tokens/stories/components/ColorSwatchGroup/ColorSwatchGroup.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { Theme } from '../../test-utils/useJsonColor';
```

Comment:
Converting doc component to use tailwind classnames and the `Text` component from `design-system-react`